src/shared/utils.tsx

```diff
@@ -16,9 +16,9 @@ export const formatTime = (oldDay: string): string => {
   return oldDay.split('.')[0]
 }
 
-export const modifyDatetime = (oldDay: string): string => {
+export const modifyDatetime = (oldDay: number): string => {
```

Comment:
Please, revert these changes since the logic of this function has been replaced with Luxon in [PR#38](https://github.com/darwinai/covidnet_ui/pull/51/files#). Also, please double check that the Luxon logic doesn't break the logic needed for this feature.

Reply:
This change was actually merged in from https://github.com/darwinai/covidnet_ui/pull/53. Will address this change when  working on implementing suggestions on that branch.